<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1345</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>581</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Cambria&quot;;</string>
    </property>
    <property name="title">
     <string>Select Location</string>
    </property>
    <widget class="QComboBox" name="stateList">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Cambria&quot;;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="cityList">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>221</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Cambria&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QListWidget" name="zipList">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>141</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Cambria&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Cambria&quot;;</string>
     </property>
     <property name="text">
      <string>State</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Cambria&quot;;</string>
     </property>
     <property name="text">
      <string>City</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Cambria&quot;;</string>
     </property>
     <property name="text">
      <string>Zip Code</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>1321</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Cambria&quot;;</string>
    </property>
    <property name="title">
     <string>Filter on Categories</string>
    </property>
    <widget class="QListWidget" name="categoryList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>351</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Cambria&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Cambria&quot;;</string>
     </property>
     <property name="text">
      <string>Select Category</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="businessTable">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>891</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Cambria&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Cambria&quot;;</string>
     </property>
     <property name="text">
      <string>Businesses</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>731</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Cambria&quot;;</string>
    </property>
    <property name="title">
     <string>Zipcode Statistics</string>
    </property>
    <widget class="QListWidget" name="businessNumber">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Cambria&quot;;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="populationNumber">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Cambria&quot;;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="averageIncome">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Cambria&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Cambria&quot;;</string>
     </property>
     <property name="text">
      <string># of Businesses:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Cambria&quot;;</string>
     </property>
     <property name="text">
      <string>Total Population:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Cambria&quot;;</string>
     </property>
     <property name="text">
      <string>Average Income:</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="categoryTable">
     <property name="geometry">
      <rect>
       <x>345</x>
       <y>40</y>
       <width>371</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Cambria&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Cambria&quot;;</string>
     </property>
     <property name="text">
      <string>Top Categories:</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="popularTable">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>591</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Cambria&quot;;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="successTable">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>540</y>
      <width>651</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Cambria&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>510</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>Popular Businesses (in zipcode)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>510</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>Successful Businesses (in zipcode)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1345</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
